Source/JavaScriptCore/runtime/InitializeThreading.cpp

```diff
@@ -119,14 +119,16 @@ void initialize()
         if (VM::isInMiniMode())
             WTF::fastEnableMiniMode();
 
-#if HAVE(MACH_EXCEPTIONS)
-        // JSLock::lock() can call registerThreadForMachExceptionHandling() which crashes if this has not been called first.
-        WTF::startMachExceptionHandlerThread();
-#endif
-        VMTraps::initializeSignals();
-#if ENABLE(WEBASSEMBLY)
-        Wasm::prepareSignalingMemory();
-#endif
+        if (Wasm::isSupported() || !Options::usePollingTraps()) {
```

Comment:
Should we make `Options::useWebAssembly()` work instead and remove `Wasm::isSupported`?
(putting `false` to `Options::useWebAssembly()` in Options.cpp when it is not supported etc., like `Options::useJIT()`).

Reply:
We already disable `Options::useWebAssembly()` in `disableAllJITOptions()`.  I think it's good to replace `Wasm::isSupported()` with `Options::useWebAssembly()`, but I'll do that in a follow up patch.